readthedocs/settings/dev.py

```diff
@@ -55,6 +56,8 @@ def DATABASES(self):  # noqa
     def LOGGING(self):  # noqa - avoid pep8 N802
         logging = super(CommunityDevSettings, self).LOGGING
         logging['formatters']['default']['format'] = '[%(asctime)s] ' + self.LOG_FORMAT
+        logging['handlers']['console']['level'] = 'DEBUG'
```

Comment:
Do we want debug logs spamming the console? @humitos and I talked on this a while back and decided that moving DEBUG level logs to a file avoids overwhelming users. I'm -0 on moving back to the console.

Reply:
OK. I found it easier to deal with, but that's fine as long as we know it is a convention. Is that what debug.log is for?